### DISCUSSION TITLE
My app is getting 404 DEPLOYMENT_NOT_FOUND error even on .vercel.app domain

### DISCUSSION THREAD

**Discussion by dianagsgs** [vercel/community#1135] [Help] [answered]
### Summary

I noticed my site https://www.labestiaradio.com/ was not loading (sometimes partially, sometimes fully) and returning 404 errors... 
I went to check if the vercel domain: https://labestia.vercel.app/ was working, and I am also getting 404 errors there.
Sometimes the page doesn't load at all and I see a DEPLOYMENT_NOT_FOUND error code. Sometimes the page loads but some of the resources don't

Any insights on why this might be happening, as far as I know we didn't change our domain config or anything else lately. This started happening today

<img width="703" alt="Captura de Pantalla 2022-12-12 a la(s) 5 36 21 p m" src="https://user-images.githubusercontent.com/80500914/207184371-2ba6c0f9-7cf7-4484-ad8c-60a0914166b7.png">

(p.s. I am new to deploying on vercel)

**Comment by swarnava** [comment]
Hi,

We can see that the website is accessible now. Are you still facing issue?

**Comment by dianagsgs** [comment]
Thanks! We are no longer seeing this issue... Are there any clues on what happened or what we can do on the application side to prevent this in the future?

**Comment by swarnava** [comment, marked as answer]
This was related to one of our internal changes. No need to take action on your side

**Comment by hammadsohail** [comment]
I'm experiencing the same on my site: https://www.japanicarparts.com/
It only appears on Google Oauth.
this is the link that i get clicking on Google Sign In, https://carparts-omega.vercel.app/api/auth/signin/google
which was vercel's domain

**Reply by swarnava** [reply]
Please create a support ticket using vercel.com/help
